metrics/profile.go

```diff
@@ -39,14 +40,15 @@ func ProfileColumns(
 			alias = "segment"
 		}
 		segmentColumns = append(segmentColumns, alias)
-		expressions = append(expressions, As(ToString(Sql(s.Field)), Identifier(alias)))
+		expressions = append(expressions, As(OptionalLengthLimit(ToString(s.Expression), segmentLengthLimit), Identifier(alias)))
```

Comment:
not sure about this ... isn't it trying cut the real column names instead of alias?

Reply:
it is about values in those columns